
# HG changeset patch
# User andrew
# Date 1266874633 0
# Node ID f25efdb55c998759ce2c1d8711011619455beaba
# Parent a3be81d385ee94edb4bffed3436fc548d68423b4
6928623: Behaviour of VERBOSE=true on langtools build
Summary: VERBOSE=true causes -diagnostics to be passed to ant rather than -debug
Reviewed-by: jjg

--- openjdk/langtools/make/Makefile	Thu Feb 18 15:41:36 2010 -0800
+++ openjdk/langtools/make/Makefile	Mon Feb 22 21:37:13 2010 +0000
@@ -70,7 +70,7 @@ endif
 endif
 
 ifdef VERBOSE
-  ANT_OPTIONS += -verbose -diagnostics
+  ANT_OPTIONS += -verbose -debug
 endif
 
 ifdef JDK_VERSION

